src/app/emails/layout.tsx

```diff
@@ -0,0 +1,44 @@
+import Image from "next/image"
+import Link from "next/link"
+
+import { buttonVariants } from "@/components/ui/button"
+import logoWhite from "@/images/logo-white.svg"
+import { cn } from "@/lib/utils"
+
+const EmailsLayout: React.FC<{ children: React.ReactNode }> = ({
+  children,
+}) => {
+  return (
+    <div className="h-full">
+      <div className="container relative hidden h-dvh flex-col items-center justify-center md:grid lg:max-w-none lg:grid-cols-2 lg:px-0">
+        <Link
+          href={"MARKETING_WEBSITE_URL"}
+          className={cn(
+            buttonVariants({ variant: "ghost" }),
+            "absolute right-4 top-4 md:right-8 md:top-8",
+          )}
+        >
+          Back
+        </Link>
+        <div className="relative hidden h-full flex-col bg-muted p-10 text-white dark:border-r lg:flex">
+          <div className="absolute inset-0 bg-zinc-900" />
+          <div className="relative z-20 flex items-center text-lg font-medium">
+            <Image src={logoWhite} alt={"logo"} width={42} className={"mr-3"} />
+            {"MARKETING_NAME"}
+          </div>
+          <div className="relative z-20 mt-auto">
+            <blockquote className="space-y-2">
+              <p className="text-lg">
+                &ldquo;Un socio é un pó come un amico.&rdquo;
```

Comment:
Weird accent here

Reply:
This is taken from previous UI state so let's get back to it at a later point if needs changing